### PR TITLE
Interpret fill color

### LINKED ISSUE #20: Exported shapes have different fill colors on export
## Problem

When you export a shape and reimport it to renew, it have the renew base color except for the original drawn color.

### PR DESCRIPTION
**Pull request by tim-kilian**
Closes #20 

## Changes
- Extended the serializer, so it reads the fill value out of the metaObject representation and adds it to the pnml

### CODE REVIEW

File: src/serializers/PnmlSerializer.js [hunk 1]
```diff
@@ -155,4 +159,51 @@ export default class PnmlSerializer {
         return labelElement;
     }
 
+    _getStyle (element) {
+        const style = { };
+        const rawStyle = element.metaObject.representation.attributes.style
+            .split(';').filter(Boolean);
```
Comment (0x4d) [inline src/serializers/PnmlSerializer.js:168]: Nice

File: src/serializers/PnmlSerializer.js [hunk 2]
```diff
@@ -155,4 +159,51 @@ export default class PnmlSerializer {
...
+        for (let i=0; i<rawStyle.length; i++) {
+            const [ key, value ] = rawStyle[i].split(':');
+            style[key.trim()] = value.trim();
+        }
+        return style;
+    }
+
+    _createFillElement (doc, element) {
+        const style = this._getStyle(element);
+        let fill = { r: 112, g: 219, b: 147 };
```
Comment (0x4d) [inline src/serializers/PnmlSerializer.js]: Why not just skip the fill element if no fill style is defined? That way the PNML is agnostic to the editor and will have that color in Renew anyways (as it is now).

Comment (tim-kilian) [reply src/serializers/PnmlSerializer.js]: You're right. Sorry about this.

**Review by 0x4d — APPROVED** [review]
Great addition to the Serializer, thanks!

**Pull request merged**